### PR TITLE
docs(GiniCaptureSDK)

### PR DESCRIPTION
**Pull request by zladzeyka**
### Description
  *  Add deprecation notice

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,19 @@
...
 [![Swift version](https://img.shields.io/badge/swift-5.0-orange.svg)]()
 
+**Deprecation Notice**
+ ----------------------
+
+ Development of the Gini Capture SDK for iOS will be continued in a 
+ [new repository](https://github.com/gini/capture-sdk-ios).
+
+ No new versions will be released from this repository and we kindly ask you to update to the 
+ [version](https://github.com/gini/capture-sdk-ios) released from the new repository.
```
Comment (a-szotyori) [inline README.md]: Version link does not point to any specific release. My intent with this was to point to the first public version released from the monorepo so they know exactly which version to update to.

The first public version of the Capture SDK swift package would be 1.1.0 ([1.0.7](https://github.com/gini/gini-capture-sdk-ios/releases) is the latest we released for CocoaPods).

File: README.md [hunk 2]
```diff
@@ -7,6 +7,19 @@
...
+
+ A few breaking changes were necessary, but these are easy to fix. You can find the steps in this 
+ [migration guide](https://developer.gini.net/gini-mobile-ios/GiniCaptureSDK/customization-guide.html).
```
Comment (a-szotyori) [inline README.md]: Link points to the customization guide instead of the migration guide.

**Review by a-szotyori — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (zladzeyka) [reply README.md]: Ok, I'll add this. 😊

File: README.md (hunk 2 above)
Comment (a-szotyori) [reply README.md]: The updated link works. 👍

File: Documentation/Sections/Documentation.md [hunk 3]
```diff
@@ -11,14 +11,14 @@
...
 
- Development of the Gini Capture SDK for iOS will be continued in a 
- [new repository](https://github.com/gini/capture-sdk-ios).
+Development of the Gini Capture SDK for iOS will be continued in 
+[new repository for Gini Capture SDK](https://github.com/gini/capture-sdk-ios) and
+[new repository for Gini Capture SDK Pinning](https://github.com/gini/capture-sdk-pinning-ios).
 
- No new versions will be released from this repository and we kindly ask you to update to the 
- [version](https://github.com/gini/capture-sdk-ios) released from the new repository.
+The first public version of the Gini Capture SDK as a swift package would be 1.1.0 ([1.0.7](https://github.com/gini/gini-capture-sdk-ios/releases/tag/1.0.7) is the latest version that we released for CocoaPods).
```
Comment (a-szotyori) [inline Documentation/Sections/Documentation.md]:
```suggestion
No new versions will be released using CocoaPods. We kindly ask you to update to the [version](https://github.com/gini/gini-mobile-ios/releases/tag/GiniCaptureSDK%3B1.1.0) available as a swift package.
```

File: Documentation/Sections/Documentation.md [hunk 4]
```diff
@@ -11,14 +11,14 @@
...
 
- A few breaking changes were necessary, but these are easy to fix. You can find the steps in this 
- [migration guide](https://developer.gini.net/gini-mobile-ios/GiniCaptureSDK/migration-guide.html).
+A few breaking changes were necessary, but these are easy to fix. You can find the steps in this 
+[migration guide](https://developer.gini.net/gini-mobile-ios/GiniCaptureSDK/migration-guide.html).
```
Comment (a-szotyori) [inline Documentation/Sections/Documentation.md]: Reading this I noticed, that the migration guide contains no breaking changes related to the integration. The only change is that swift package manager has to be used.
```suggestion
You can find the steps needed to switch to swift package manager in this [migration guide](https://developer.gini.net/gini-mobile-ios/GiniCaptureSDK/migration-guide.html).
```

**Review by a-szotyori — COMMENTED** [review]
I left two suggestions. What do you think?

**Review by a-szotyori — APPROVED** [review]
Thank you! Looks good now 😁!

**Pull request merged**